firestore/conformance_test.go

```diff
@@ -191,7 +190,7 @@ func runTest(test *pb.Test, c *Client, srv *mockServer) error {
 		got, checkErr := q.toProto()
 		if err := check(checkErr, typedTestcase.Query.IsError); err == nil && checkErr == nil {
 			if want := typedTestcase.Query.Query; !proto.Equal(got, want) {
-				return fmt.Errorf("got:  %s\nwant: %s", proto.MarshalTextString(got), proto.MarshalTextString(want))
+				return fmt.Errorf("got:  %s\nwant: %s", got, want)
```

Comment:
ditto re: using `prototext` marshaler...

Reply:
resolving as a dupe, there are multiple instances and I don't  want to comment on each.